cranelift/jit/src/compiled_blob.rs

```diff
@@ -38,39 +38,39 @@ impl CompiledBlob {
             match kind {
                 Reloc::Abs4 => {
                     let base = get_address(name);
-                    let what = unsafe { base.offset(isize::try_from(addend).unwrap()) };
+                    let what = base.wrapping_offset(isize::try_from(addend).unwrap());
```

Comment:
This would loose optimizations and isn't more correct. `.offset()` is unsafe as it is UB to go out of bounds, but if the calculation goes out of bounds, this crate would produce UB anyway, except now sanitizers no longer know that going out of bounds is invalid and thus wouldn't trigger at the root cause of the UB.